database.c

```diff
@@ -180,14 +180,18 @@ static int do_sleep(struct timeval *delta) {
 #endif
 }
 
-static void print_replay_statistics() {
+static void print_replay_statistics(int dry_run) {
```

Comment:
I don't think we need the `dry_run` parameter.
All it does is add `(dry run)` to the output, and while I admit that it serves a certain documentation value, the absence of any runtime statistics will probably be documentation enough.